openslides_backend/action/actions/meeting/import_.py

```diff
@@ -131,11 +169,11 @@ def check_usernames_and_generate_new_ones(self, json_data: Dict[str, Any]) -> No
         used_usernames = set()
```

Comment:
Can you unify this method with the one in `user.create` so the behaviour does not diverge again? If you want, we can also do this in a seperate issue to not blow up this one even more

Reply:
done